histomicstk/cli/NucleiDetection/NucleiDetection.py

```diff
@@ -110,6 +125,10 @@ def detect_tile_nuclei(tile_info, args, src_mu_lab=None,
     flag_nuclei_found = np.any(im_nuclei_seg_mask)
 
     if flag_nuclei_found:
+        if args.nuclei_annotation_format == 'bbox' and args.remove_overlapping_nuclei_segmentation:
```

Comment:
This might be clearer to do
```
if flag_nuclei_found:
    format = args.nuclei_annotation_format
    if args.nuclei_annotation_format == 'bbox' and args.remove_overlapping_nuclei_segmentation:
        format = 'boundary'
    nuclei_annot_list = cli_utils.create_tile_nuclei_annotations(im_nuclei_seg_mask, tile_info, format)
```

rather than have two return paths.